tests/test_transcription.py

```diff
@@ -0,0 +1,201 @@
+# CREATED: 2/9/16 2:27 PM by Justin Salamon <justin.salamon@nyu.edu>
+
+import mir_eval
+import numpy as np
+import glob
+import json
+from nose.tools import raises
+import warnings
+
+A_TOL = 1e-12
+
+# Path to the fixture files
+REF_GLOB = 'tests/data/transcription/ref*.txt'
+EST_GLOB = 'tests/data/transcription/est*.txt'
+SCORES_GLOB = 'tests/data/transcription/output*.json'
+
+REF = np.array([
+    [0.100, 0.300, 220.000],
+    [0.300, 0.400, 246.942],
+    [0.500, 0.600, 277.183],
+    [0.550, 0.650, 293.665]])
+
+EST = np.array([
+    [0.120,   0.290,   225.000],
+    [0.300,   0.340,   246.942],
+    [0.500,   0.600,   500.000],
+    [0.550,   0.600,   293.665],
+    [0.560,   0.650,   293.665]])
+
+SCORES = {
+    "Precision": 0.4,
+    "Recall": 0.5,
+    "F-measure": 0.4444444444444445,
+    "Precision_no_offset": 0.6,
+    "Recall_no_offset": 0.75,
+    "F-measure_no_offset": 0.6666666666666665
+}
+
+
+def test_match_notes():
+
+    ref_int, ref_pitch = REF[:, :2], REF[:, 2]
+    est_int, est_pitch = EST[:, :2], EST[:, 2]
+
+    matching = \
+        mir_eval.transcription.match_notes(ref_int, ref_pitch, est_int,
+                                           est_pitch)
+
+    assert matching == [(0, 0), (3, 3)]
+
+    matching = \
+        mir_eval.transcription.match_notes(ref_int, ref_pitch, est_int,
+                                           est_pitch, offset_ratio=None)
+
+    assert matching == [(0, 0), (1, 1), (3, 3)]
+
+
+def test_match_notes_strict():
+
+    ref_int, ref_pitch = np.array([[0, 1]]), np.array([100])
+    est_int, est_pitch = np.array([[0.05, 1]]), np.array([100])
+
+    matching = \
+        mir_eval.transcription.match_notes(ref_int, ref_pitch, est_int,
+                                           est_pitch, strict=True)
+
+    assert matching == []
+
+
+def test_precision_recall_f1():
+
+    # load test data
+    ref_int, ref_pitch = REF[:, :2], REF[:, 2]
+    est_int, est_pitch = EST[:, :2], EST[:, 2]
+
+    precision, recall, f_measure = \
```

Comment:
Nitpick: https://docs.python.org/2/howto/doanddont.html#using-backslash-to-continue-statements